MEP_Engine/Query/Geometry.cs

```diff
@@ -46,5 +47,25 @@ public static IGeometry Geometry(this IFlow flowObj)
         }
 
         /***************************************************/
+
+        public static List<Line> Geometry(this Fitting fitting)
+        {
+            List<Line> result = new List<Line>();
+            if (fitting.ConnectionsLocation.Count == 2)
+            {
+                result.Add(BH.Engine.Geometry.Create.Line(fitting.ConnectionsLocation[0],fitting.ConnectionsLocation[1]));
+            }
+            else
+            {
+                foreach (Point point in fitting.ConnectionsLocation)
+                {
+                    result.Add(BH.Engine.Geometry.Create.Line(fitting.Location,point));
```

Comment:
```suggestion
                    result.Add(BH.Engine.Geometry.Create.Line(fitting.Location, point));
```

If a fitting has 8 connection points, this would create a spoked geometry all feeding through the location point, rather than a continuous line. Is that the desired functionality?

Reply:
Indeed that is the purpose, although I don't we have fittings with more than 4 connections like a cross. @michaelhoehn @kayleighhoude what do you think?